### ISSUE TITLE
Bug on pod storage handler

### ISSUE THREAD

**Issue by fincamd** [Arquisoft/viade_en3a#106]
I have come across an error given by the pod storage handler module. It raises this mentioned error when the folder that holds the routes inside the user's POD does not exist.

I think it would be necessary to check if such folder exists before trying to access its contents. Maybe then, with that information we can warn the user, but most likely we will be able to make our app more robust to failure.

Do you agree?

**Comment by oscar134** [comment]
Seems a good idea checking for the existance of the folder

**Comment by Drastur** [comment]
It should be possible to check for its existance as @oscar134 said, and in case it doesn't exist just create the folder /viade/routes in the pod.

**Comment by fincamd** [comment]
Yeah, the thing of creating the folder inside user's pod is a great idea. I agree on that too

**Comment by jesQM** [comment]
This error has now been fixed.
My bad, everytime I tested the upload/download I had the folders created.

So:
* If the folders are not present, they are created and the routes returned is indeed an empty array (as there are no routes in the pod).
* Some errors seem to be written onto the console (like 404 folder not found, dont really know why, but will look into it, however, the PR will be merged so u guys have stuff to work with)-
* A deleteAll method was added to clear any viade stuff of a pod (specially for testing purposes), as all folders gets erased, well... it kinda trows exceptions too 💩
